README.md

```diff
@@ -1,11 +1,95 @@
-# Airflow conncetor for Ocean For Apache Spark
+# Airflow connector for Ocean Apache Spark
 
-<!-- TODO(crezvoy): long description -->
+An Airflow plugin and provider to launch and monitor Spark
+applications on the [Ocean for
+Spark](https://spot.io/products/ocean-apache-spark/).
 
-# Compatibility
+## Compatibility
 
-# Installation
+`ocean-spark-airflow-provider` is compatible with both airflow 1 and
+airflow 2. it is detected as an airflow plugin by airlfow 1 and up,
```

Comment:
```suggestion
airflow 2. it is detected as an airflow plugin by airflow 1 and up,
```
Why "and up"? Do you mean it is detected as a plugin and as a provider by Airflow 2?

Reply:
Also, please add a capital "A" to all instances of Airflow in the text

Reply:
> Why "and up"? Do you mean it is detected as a plugin and as a provider by Airflow 2?

Yes, its is detected both as a plugin and a provider by Airflow 2. Airflow does not seem to mind, I think the only consequence is that one can import the operator from `airflow.operators.` (plugin) and from `ocean_spark.operators` (provider).